pkg/sql/schemachanger/scpb/state.go

```diff
@@ -112,27 +112,35 @@ func (m *DescriptorState) Clone() *DescriptorState {
 // MakeCurrentStateFromDescriptors constructs a CurrentState object from a
 // slice of DescriptorState object from which the current state has been
 // decomposed.
-func MakeCurrentStateFromDescriptors(descriptorStates []*DescriptorState) (CurrentState, error) {
+func MakeCurrentStateFromDescriptors(
+	descriptorStates []*DescriptorState,
+) (_ CurrentState, nonCancelable bool, _ error) {
```

Comment:
Consider adding `NonCancelable` to `CurrentState`? I don't quite get why it's kept separate from it.